.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Motivation and Context
+
+<!--
+  Why is this change required? What problem does it solve?
+  This can be omitted if a linked issue is provided 
+-->
+
+## Description
+
+<!-- Describe your changes -->
+
+## Test Plan
+
+<!-- Please describe in detail how you tested your changes. -->
```

Comment:
How have you tested your changes? Are there additional steps required to deploy this change?

Reply:
Like it, refactored with https://github.com/coordinape/coordinape/pull/913/commits/b0578b981b9d4364be55bd5fb56aea17e551a26b